DllImportGenerator/DllImportGenerator/Marshalling/CustomNativeTypeMarshaler.cs

```diff
@@ -101,16 +101,7 @@ public IEnumerable<StatementSyntax> Generate(TypePositionInfo info, StubCodeCont
             }
             switch (context.CurrentStage)
             {
-                case StubCodeContext.Stage.Setup:
-                    if (!_marshalerTypePinnable || info.IsByRef && info.RefKind != RefKind.In)
-                    {
-                        yield return LocalDeclarationStatement(
-                        VariableDeclaration(
-                            _nativeTypeSyntax,
-                            SingletonSeparatedList(
-                                VariableDeclarator(nativeIdentifier))));
-                    }
-                    
+                case StubCodeContext.Stage.Setup:                    
```

Comment:
Oops. Would this even build? What are the semantics when this is removed?

Reply:
Yes. In Elinor's work, we've moved the `nativeIdentifier` to be automatically declared when `UsesNativeIdentifier` is true. This ensures that the implementation of `UsesNativeIdentifier` is accurate (and found the bug that I fixed in the other commit I pushed).

Reply:
A non-cased statement (i.e. ` if (_useValueProperty) ...`) is permitted in a `switch` statement is the main tenor of my question. What does it mean?

Reply:
Handled offline - no issue here. Aaron's diff was confusing.

Reply:
@jkoritzinsky Sorry.